features/background_requires_scenario.feature

```diff
@@ -53,5 +54,5 @@ Feature: Background Requires Scenario
     When I run `ruby lint.rb`
     Then it should pass with exactly:
       """
-
+      There are no issues
```

Comment:
I see you already have a --verbose option that defaults to true.  What rubocop does is treats the report output as separate by writing it to a file.  I'm happy to do something like that but I want your feedback on how to move forward.  I agree that this is fragile but it's also pretty easy to change so let me know what you want to do here.

For us, this is less about trusting the return code.  We want positive confirmation that gherkin_lint ran and everything was cool (or not)